tor2web/t2w.py

```diff
@@ -678,14 +678,17 @@ def process_request(self, req):
         self.obj.address = "http://" + self.obj.onion + self.obj.uri
 
         self.obj.headers = req.headers
-
+        self.proto = b'http' if config.transport == 'HTTP' else b'https'
```

Comment:
ok for this patch but i want to do this thing that is going to be used in multiple parts only once storing in a variable that would be accessible by all the app.